packages/server/lib/controllers/auth/postApiKey.ts

```diff
@@ -230,6 +232,8 @@ export const postPublicApiKeyAuthorization = asyncWrapper<PostPublicApiKeyAuthor
             metadata: { providerConfigKey, connectionId }
         });
 
+        metrics.increment(metrics.Types.AUTH_FAILURE, 1, { auth_mode: 'API_KEY' });
+
```

Comment:
Could we have the metrics in `AccessMiddleware` instead of having to repeat it for every single auth type

Reply:
access middleware is for global authentication, but I see your point

Reply:
actually it's annoying because we don't store the auth_mode, going to keep it like this until better